packages/functional-tests/lib/fixtures/standard.ts

```diff
@@ -150,48 +163,43 @@ export const test = base.extend<TestOptions, WorkerOptions>({
     });
   },
 
-  emailOptions: [{ prefix: '', PASSWORD }], // Default options for the fixture
+  emailOptions: [{ prefix: SIGNIN_EMAIL_PREFIX, password: PASSWORD }], // Default options for the fixture
 
   emails: async (
     { target, pages: { login }, emailOptions },
     use
   ): Promise<void> => {
-    const emails = await Promise.all(
-      emailOptions.map(async (emailOptions) =>
-        login.createEmail(emailOptions.prefix ?? '')
-      )
-    );
+    const emails = emailOptions.map((emailOptions) => {
+      return (
+        emailOptions.prefix.replace('{id}', Math.random() + '') +
+        '@restmail.net'
+      );
+    });
     await login.clearCache(); // Clear cache for each email
 
     // Pass the generated email to the test along with the password
     await use(emails);
+
     for (const [index, email] of emails.entries()) {
-      const emailExists = await target.auth.accountStatusByEmail(email);
-      if (emailExists.exists) {
-        const x = emailOptions[index];
-        await teardownEmail(target, email, x.NEW_PASSWORD ?? x.PASSWORD);
-      }
+      await teardownEmail(target, email, emailOptions[index]);
```

Comment:
I might be missing something, but it doesn't look like `password` via `emailOptions` is being used; so I assume it's just being passed here along w/ `prefix`. 
Not sure it'd be any easier to default the `password` to `PASSWORD` const around here so it doesn't need to be passed in explicitly each time. But now that I type that, it sounds unrelated to the PR and just a future enhancement idea (something like `{... password: PASSWORD, ...emailOptions[index]}`, which means _in theory_ we could just use this:
```js
{ prefix: SIGNIN_EMAIL_PREFIX }
// or this, if you want to override default password.
{ prefix: SIGNIN_EMAIL_PREFIX, password: NEW_PASSWORD }
```

Reply:
I thought about making password optional, but in most places the tests import the default PASSWORD anyway and by forcing it's definition in the test classes, developers can better connect if there is a mismatch.